src/main/perl/CLI.pm

```diff
@@ -505,10 +504,6 @@ sub action
         chroot($self->option("chroot")) or die "Unable to chroot to ", $self->option("chroot");
     }
 
-    if(! chdir($NCD_CHDIR)) {
```

Comment:
why did you remove this?

Reply:
in particular, does chroot still work? the tempdir is created during import, does it also exist in the chroot environment?

Reply:
As summarized above it seemed redundant to me to do the chdir in `NCD::CLI` because it happens before each module execution. Is it needed?

For chroot - I think you are right and I need to make adjustments to the code to create the directory in the chroot.

Reply:
main reason is to do this asap in the commandflow. but i'm not sure it has that much advatange.
as along as you define and the create the tempdir only once.